includes/admin/llms.functions.admin.php

```diff
@@ -288,3 +288,113 @@ function llms_merge_code_button( $target = 'content', $echo = true, $codes = arr
 	return $html;
 
 }
+
+/**
+ * Deletes all pending orders after held duration.
+ *
+ * @since [version]
+ *
+ * @return null
+ */
+function llms_delete_pending_orders() {
+
+	$days = get_option( 'lifterlms_pending_orders_deletion' );
+
+	if ( ! $days ) {
+		return;
+	}
+
+	// Get all LLMS pending orders of custom post `llms_order`.
+	$orders = get_posts(
```

Comment:
can we create single query here? like query all the `llms_order`s with `post_status` equal to `llms-pending` created more than `$days` ago?
E.g. using the [date_query/date params](url)? https://wordpress.stackexchange.com/questions/310394/how-to-get-posts-on-a-specific-date-wp-query
Or https://tommcfarlin.com/search-posts-by-date-range/